scss/style.scss

```
@@ -1,86 +1,139 @@
+            margin-left: 20px;
+        }
+        &__link {
             display: flex;
-            justify-content: flex-end;
+            flex-flow: row nowrap;
+            align-items: center;
+            text-decoration: none;
+            font-size: $txt-size;
+            color: $bg-main;
         }
-        &:first-child {
-            margin-left: 36px;
+        &__span {
+            font-size: 12px;
+            padding-left: 5px;
         }
-        &:last-child {
-            margin-right: 36px;
+        &__img-icon {
+            width: 24px;
+            height: 24px;
+        }
+        &__search-form {
+            height: 24px;
+            width: 152px;
+            padding-left: 20px;
+            &__input {
+                width: 152px;
+                height: 24px;
+                background-color: $bg-input;
+                border: none;
+                border-radius: 8px;
+            }
         }
     }
-    li {
-        list-style-type: none;
-        margin-left: 20px;
+    
+    .wrapper {
+        display: flex;
+        flex-direction: row;
+        height: 90%;
     }
-    a {
-        text-decoration: none;
-        padding: 0;
-        .value {
-            font-size: 16px;
-            color: white;
-            text-align: center;
-            line-height: 1.8;
+    
+    .toolbar {
+        width: 30%;
+        height: 100%;
+        background-color: $bg-toolbar;
+        border: $border-size;
+    }
+    
+    .board {
+        width: 100%;
+        display: flex;
+        flex-direction: column;
+        &__header {
+            height: 56px;
+            margin-bottom: $grid-step;
+        }
+        &__title {
+            font-weight: 300;
+        }
+        &__bottom {
+            height: 100%;
+            display: flex;
+            flex-direction: row;
         }
     }
-    img {
-        width: 24px;
-        height: 24px;
+    
+    .column {
+        flex-basis: 40%;
+        background-color: $bg-main;
     }
-    .header__search-form {
-        height: 24px;
-        width: 150px;
-        padding-left: 20px;
-        .header__search-form__input {
-            padding: 0;
-            width: 150px;
-            height: 24px;
-            background-color: #e36f66;
-            border: none;
-            border-radius: 4%;
+    
+    .subtitle {
+        &__text {
+            padding: $grid-step 16px;
+            font-size: $txt-size;
+            font-weight: 600;
+        }
+        &__img-icon {
+            width: 16px;
+            height: 16px;
+            margin-left: $grid-step;
         }
     }
-}
-
-.wrapper {
-    display: flex;
-    justify-content: flex-start;
-    height: 100%;
-}
-
-.menu {
-    width: 30%;
-    background-color: #fbfafb;
-    border: solid 1px;
-    border-color: #e8e8e8;
-}
-
-.content {
-    width: 30%;
-    background-color: white;
-}
\ No newline at end of file
+    
+    .card {
+        border: $border-size;
+        border-radius: 8px;
```

Обычно во всем проекте эта величина одинаковая, поэтому в переменную вынеси

```suggestion
        border-radius: $border-radius;
```